26/upgrade.html

```diff
@@ -19,6 +19,50 @@
 
 <script id="upgrade-template" type="text/x-handlebars-template">
 
+<h4><a id="upgrade_2_6_0" href="#upgrade_2_6_0">Upgrading to 2.6.0 from any version 0.8.x through 2.5.x</a></h4>
+
+<p><b>If you are upgrading from a version prior to 2.1.x, please see the note below about the change to the schema used to store consumer offsets.
+    Once you have changed the inter.broker.protocol.version to the latest version, it will not be possible to downgrade to a version prior to 2.1.</b></p>
+
+<p><b>For a rolling upgrade:</b></p>
+
+<ol>
+    <li> Update server.properties on all brokers and add the following properties. <code>CURRENT_PROTOCOL_VERSION</code> refers to the 
+        <a href="/26/documentation/#inter.broker.protocol.version">inter-broker protocol version</a> you are upgrading from. 
+        <code>CURRENT_MESSAGE_FORMAT_VERSION</code> refers to the <a href="/26/documentation/#log.message.format.version">log message format version</a>
+        currently in use. If you have previously overridden the message format version, you should keep its current value.
+        Alternatively, if you are upgrading from a version prior
+        to 0.11.0.x, then <code>CURRENT_MESSAGE_FORMAT_VERSION</code> should be set to match <code>CURRENT_PROTOCOL_VERSION</code>.
+        <ul>
+            <li>inter.broker.protocol.version=CURRENT_PROTOCOL_VERSION</li>
+            <li>log.message.format.version=CURRENT_MESSAGE_FORMAT_VERSION  (See <a href="#upgrade_10_performance_impact">potential performance impact
+                following the upgrade</a> for the details on what this configuration does.)</li>
+        </ul>
+        If you are upgrading from version 0.11.0.x or above, and you have not overridden the message format, then you only need to override
+        the inter-broker protocol version.
+        <ul>
+            <li>inter.broker.protocol.version=CURRENT_PROTOCOL_VERSION</li>
+        </ul>
+    </li>
```

Comment:
This section contains a few of the changes, but they're harder to see:
* a link to the `inter.broker.protocol.version` config documentation
* renamed from `CURRENT_KAFKA_VERSION` to `CURRENT_PROTOCOL_VERSION` since the value is no longer just the version
* no longer listing potential values (the old values were not correct anyway)

Reply:
These changes were reverted based upon suggestions (offline).